fdm-core/src/field.ts

```diff
@@ -166,6 +166,7 @@ export async function getField(
                 b_centroid_x: sql<number>`ST_X(ST_Centroid(b_geometry))`,
                 b_centroid_y: sql<number>`ST_Y(ST_Centroid(b_geometry))`,
                 b_area: sql<number>`ROUND((ST_Area(b_geometry::geography)/10000)::NUMERIC, 2)::FLOAT`,
+                b_perimeter: sql<number>`ROUND((ST_Length(ST_ExteriorRing(b_geometry)::geography))::NUMERIC, 2)::FLOAT`,
```

Comment:
so, perimeter is in unit meter whereas area is in unit hectare. is that by purpose?

Reply:
i see that you correct for the unit difference in determineIfFieldIsProductiveByShape, so that is fine.